docs/carbon-as-spark-datasource-guide.md

```diff
@@ -44,22 +45,20 @@ Carbon table can be created with spark's datasource DDL syntax as follows.
 | table_blocksize | 1024 | Size of blocks to write onto hdfs |
```

Comment:
add sentence to refer to the original definition and explanation along with link to it.